api/grpcserver_test.go

```diff
@@ -1292,13 +1297,17 @@ func TestGrpcServer_GetChainMeta(t *testing.T) {
 			testutil.CleanupPath(t, bfIndexFile)
 		}()
 		if pol != nil {
-			require.NoError(pol.ForceRegister(svr.core.registry))
+			require.NoError(pol.ForceRegister(svr.core.Registry()))
 		}
 		if test.emptyChain {
 			mbc := mock_blockchain.NewMockBlockchain(ctrl)
 			mbc.EXPECT().TipHeight().Return(uint64(0)).Times(1)
 			mbc.EXPECT().ChainID().Return(uint32(1)).Times(1)
-			svr.core.bc = mbc
+			coreService, ok := svr.core.(*coreService)
+			if !ok {
+				require.Error(err)
+			}
+			coreService.bc = mbc
```

Comment:
add a TODO: create a service with empty blockchain to do `emptyChain` related test.

Reply:
yes